package-lock.json

```diff
@@ -37432,4 +37432,4 @@
 			"dev": true
 		}
 	}
-}
+}
```

Comment:
There are still some package-lock.json changes, please remove these changes.

Reply:
@yash-rajpal  I'm not sure why is this change shown even though I didn't commit it. If you see it says, the brace deleted from 37435 and added again to the same line.
This is a problem I faced in my other commit too.